cdap-kms/pom.xml

```diff
@@ -25,7 +25,7 @@
     <version>6.9.0-SNAPSHOT</version>
   </parent>
   <properties>
-    <hadoop.common.version>2.6.0</hadoop.common.version>
+    <hadoop.common.version>2.6.5</hadoop.common.version>
```

Comment:
nit: We should be able to remove this property and let it inherit version from parent pom.

Reply:
Done